.travis.yml

```diff
@@ -5,23 +5,7 @@ matrix:
     - script:
         - bin/fetch-configlet
         - bin/configlet lint .
-        - docker run -v `pwd`:/swift swift:4.0.2 bash -c "cd /swift && ./xswift-test-spm"
-      env: JOB=Linux40
-      sudo: required
-      services: docker
-    - script:
-        - bin/fetch-configlet
-        - bin/configlet lint .
-        - docker run -v `pwd`:/swift norionomura/swift:swift-4.1-branch bash -c "cd /swift && ./xswift-test-spm"
-      env: JOB=Linux41x
-      sudo: required
-      services: docker
-    - script:
-        - bin/fetch-configlet
```

Comment:
We need these:
```
        - bin/fetch-configlet
        - bin/configlet lint .
```
They check the validity of each exercise config

Reply:
We have these in the `.travis.yml` file

Full contents of [`.travis.yml`](https://github.com/bhargavg/swift/blob/chore%2Fupgrade-swiftpm-4/.travis.yml):

```YAML
language: generic

matrix:
  include:
    - script:
        - bin/fetch-configlet
        - bin/configlet lint .
        - docker run -v `pwd`:/swift swift:4.2 bash -c "cd /swift && ./xswift-test-spm"
      env: JOB=Linux42x
      sudo: required
      services: docker
```